Lib/asyncio/helpers.py

```diff
@@ -0,0 +1,142 @@
+from contextlib import suppress
```

Comment:
I'm usually against using module names like "helpers" or "utils", as it's impossible to know what's in them by just looking at the name. I propose to call this module "staggered.py".